core/ir/x86/decode_fast.c

```diff
@@ -1359,6 +1359,9 @@ decode_cti(dcontext_t *dcontext, byte *pc, instr_t *instr)
                 /* VEX 2-byte prefix implies 0x0f opcode */
                 byte0 = 0x0f;
                 byte1 = *(pc + prefixes);
+                /* There are no prefixes after vex. */
+                pc = start_pc + prefixes;
+                i = prefixes;
```

Comment:
Thanks for looking into this.  The fix makes sense.  I'm wondering why none of the api.ir tests hit this -- I guess none happen to have the right byte sequence.  Could we add a test that hits this?  Even just one case in api.ir would be good to have a regression test.

Reply:
Nothing in apir.ir or in the binutils tests has the right byte sequence (I tried adding a decode_cti check to the latter but it doesn't trip).

Reply:
Add a vandps entry to api.ir with the same operands as the culprit in #4652 (looks like i#4652 can go in the title and have 
Fixes here)?  `vandps 0xa0c1d68(%rip),%ymm7,%ymm5`

Reply:
Alright that should do it.